src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -689,6 +696,18 @@ rpm_ostree_compose_context_new (const char    *treefile_pathstr,
                                error))
     return FALSE;
 
```

Comment:
Seems like we should error out somewhere around here if layers are provided in non-unified-mode? (I mean, we're trying to get rid of it, but until then, it's clearer to not mix the two paths.)